### DISCUSSION TITLE
Custom F/W On UDM base?

### DISCUSSION THREAD

**Discussion by redpoisonv3** [fabianishere/udm-kernel-tools#69] [General]
Just wondering if the firmware will work on the udm base also or is it only for the pro?

**Comment by fabianishere** [comment]
This project should run on both the UDM Base and the UDM Pro. However, you might run into trouble with the WiFi drivers on the UDMB (see #67).

**Reply by redpoisonv3** [reply]
so only the built in wifi and not my attached aps?, i dont use the wifi on the base since i have some U6 pros.

**Reply by fabianishere** [reply]
Yes, only the WiFi drivers for the built-in WiFi chip. External APs are not affected.
